canonical-websites/www.canonical.com
# updated news and resources block to use new, non-google feed api, javascript

# Details
- card: https://trello.com/c/QxTSBfNG
## done
- removed the old google feed api code and replaced it with the newer jquery version
- also removed factsheets as they aren't on insights any more
## qa
-  go to www.canonical.com/cloud and see that the news and resources block have articles just like the live website (only you cannot as the feeds are CORs blocked, you could use test feeds from http://www.stmgrts.org.uk/taleo/)
